src/App.js

```diff
@@ -3,12 +3,36 @@ import "./App.css"
 import LightSwitch from "./components/LightSwitch"
 
 const App = () => {
-  const [lightSwitchStatus, setLightSwitchStatus] = useState("off")
+  const [lightSwitchStatus, setLightSwitchStatus] = useState('off')
 
+  // This code block toggles the light switch. It changes the setLightSwitchStatus variable from on to off. 
+
+  const toggleLightSwitch = () => {
+    if(lightSwitchStatus === 'off') {setLightSwitchStatus('on')
+     } else if (lightSwitchStatus === 'on') {
+    setLightSwitchStatus('off')  
+     }
+    }
```

Comment:
A lot of us moved this function into the LightSwitch.js file to keep all of the related material in a single component. When you do this you don't end up needing to use Props at all.